frontend/containers/forms/project-gallery/project-gallery-image/types.ts

```diff
@@ -0,0 +1,30 @@
+import { UseFormRegisterReturn, RegisterOptions, FieldPath } from 'react-hook-form';
+
+export type ProjectGalleryImageType = {
+  /** Image ID */
+  id: string;
+  /** Title of the image */
+  title: string;
```

Comment:
Since the images will be uploaded by the user, we won't get any titles, right? Should we make it optional?

Reply:
We should have some auto generated titles IMHO, even if just "Image 1, Image 2" (`image {index + 1}` perhaps), so it's more obvious what the user is selecting. As it is it'll be the responsibility of the wrapper to set it. 

One thing I've left in the open, is that there may be value into having a HOC wrapping both the (future) uploader and this gallery, so that HOC could have the responsibility to decide image titles / tell the SR what's being selected.